File: Leetcode/swapNodes.py
class Solutin:
    def swapPairs(self,head);
        if head is not None and head.next is not None:
            head.next.val,head.val=head.val,head.next.val
            head.next.next = self.swapPairs(head.next.next)
        return head
        

File: Leetcode/swapNodes.js
var swapPairs = function (head) {
  if (head.next === null) {
    return head;
  }
  if (head.next !== null && head !== null) {
    var p1 = head.val;
    head.val = head.next.valhead.next.cal = p1;
    if (head.next.next !== null) {
      head.next.next = swapPairs(head.next.next);
    }
  }
  return head;
};
